### DISCUSSION TITLE
Download chapter by chapter

### DISCUSSION THREAD

**Discussion by Pookybear123** [lncrawl/lightnovel-crawler#2280] [Q&A]
Is it possible to dowload one chapter at a time. There are websites where every chapter has a password and if i use the url for the table of contents it would download blank pages since it cant access chapters with password. I want to download using the url of every chapter i access.

**Comment by tward1505** [comment]
Ok, I will assume you have the standalone exe instead of the installed version (which these steps will definitely work for), and these instructions are for windows, but I'm sure mac has a similar enough way to do it.

Open cmd prompt (windows button/search>"cmd")
navigate to folder your lncrawl.exe is in. (to make things simple let's assume it's in downloads, which means you only have to type "cd downloads")
Now, I would recommend filling in the [brackets] with the correct info, first I will post a working so you can reference what I'm talking about.

`lncrawl -s https://boxnovel.com/novel/heavenly-dao-rankings-i-am-exposed-as-the-sword-god/ -o "E:\test" --chapters https://boxnovel.com/novel/heavenly-dao-rankings-i-am-exposed-as-the-sword-god/chapter-310/`

Now the break down for you to fill in with the info you want, and paste into the command prompt (right click is paste for me)

lncrawl -s [table of contents url here] -o "[where to put the file on your pc here]" --chapters [url for chapter here]

It seems you can reference multiple chapter urls separated by a space(?), but I didn't test, especially since it sounds like you are dealing with time sensitive urls.
